engine.js

```diff
@@ -1083,32 +1083,40 @@ var accumulateResult = function(ifResult, thenResult) {
     };
 
     HMDAEngine.isValidNumHomePurchaseLoans = function(hmdaFile) {
-        var count = 0;
+        var count = 0,
+            countSold = 0;
         _.each(hmdaFile.loanApplicationRegisters, function(element, index, next) {
-            if (element.loanPurpose === '1' && _.contains(['1', '6'], element.actionTaken) && _.contains(['1', '2'], element.propertyType) && element.purchaserType !== '0') {
```

Comment:
This pull references the addition of a count of sold loans.. no where does it mention that another clarification about the purchaserType. Is this correct?